### ISSUE TITLE
CogvideoX 1.5 5b uses  more video memory than Alibaba Fun 1.15b, causing 22GB to fail to run

### ISSUE THREAD

**Issue by littleyeson** [kijai/ComfyUI-CogVideoXWrapper#290]
There 768*768 32freams cannot run in 1.5-5b mode.But In  Alibaba Fun 1.15b can run 1280*720 49freams in 2080ti 22G.But I run too slowly the upgrade 8 release in Alibaba Fun 1.15b and Tora 1.0-5B
![微信图片_20241122010834](https://github.com/user-attachments/assets/4f78dfcb-8ff3-4709-8f40-42f329c56d7f)
![微信截图_20241122011509](https://github.com/user-attachments/assets/0f7f7c8a-d758-4ced-a96b-2b78b40d322e)
![微信截图_20241123215955](https://github.com/user-attachments/assets/78729eca-a65c-4b61-847b-6ca2feac1aaf)

**Comment by kijai** [comment]
What is the Fun workflow supposed to do? It's setup wrong for img2video at least.

**Comment by littleyeson** [comment]
![微信截图_20241124104436](https://github.com/user-attachments/assets/0fc054e8-4d2b-4f60-8c83-899e8f5f66ba)
I uesd the i2v workflow in example,still not work

**Comment by kijai** [comment]
Your GPU doesn't support bf16 and 1.5 only runs on that.

**Comment by littleyeson** [comment]
> What is the Fun workflow supposed to do? It's setup wrong for img2video at least.

Before the 8 release update, The same workflow was running normally and could generate images

**Comment by kijai** [comment]
> > What is the Fun workflow supposed to do? It's setup wrong for img2video at least.
> 
> Before the 8 release update, The same workflow was running normally and could generate images

Yes, like the update notice explains old workflows will not work as they are, the nodes and their inputs have changed a lot, looking at your screenshot they are wired wrong now. You should refer to the new examples on how they should be wired.
